src/reaper_osara.cpp

```diff
@@ -4238,6 +4255,16 @@ bool handlePostCommand(int section, int command, int val=0, int valHw=0,
 			isHandlingCommand = false;
 			return true;
 		}
+	} else if(section == MEDIA_EXPLORER_SECTION) {
+		const auto it = MExplorerPostCommands.find(command);
+		if(it != MExplorerPostCommands.end()){
+			isHandlingCommand = true;
+			SendMessage(hwnd, WM_COMMAND, command, 0);
```

Comment:
Nice. I didn't realise you could trigger actions with WM_COMMAND, but it makes sense. I really thought there was an API function to run a command in an arbitrary section, but apparently there isn't!